shared/src/business/useCases/updatePetitionDetailsInteractor.js

```diff
@@ -32,13 +32,15 @@ exports.updatePetitionDetailsInteractor = async (
 
   const editableFields = {
     caseType: petitionDetails.caseType,
+    hasVerifiedIrsNotice: petitionDetails.hasVerifiedIrsNotice,
     irsNoticeDate: petitionDetails.irsNoticeDate,
     petitionPaymentDate: petitionDetails.petitionPaymentDate,
     petitionPaymentMethod: petitionDetails.petitionPaymentMethod,
     petitionPaymentStatus: petitionDetails.petitionPaymentStatus,
     petitionPaymentWaivedDate: petitionDetails.petitionPaymentWaivedDate,
     preferredTrialCity: petitionDetails.preferredTrialCity,
     procedureType: petitionDetails.procedureType,
+    statistics: petitionDetails.statistics,
```

Comment:
Nit: is it worth it to add a test around the editable fields being updated on the case?

Reply:
Are you thinking something like verifying that ONLY editable fields are being consumed?

Reply:
I think so!

Reply:
Added a test that verifies rogue fields are not updated!